ActorRepositoryInterface.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace XApi\Repository\Api;
+
+use Xabbuh\XApi\Common\Exception\NotFoundException;
+use Xabbuh\XApi\Model\Agent;
+
+/**
+ * Public API of an Experience API (xAPI) {@link Actor} repository.
+ *
+ * @author Jérôme Parmentier <jerome.parmentier@acensi.fr>
+ */
+interface ActorRepositoryInterface
+{
+    /**
+     * Finds a collection of {@link Agent agents} related to one {@link Agent agent}.
+     *
+     * @param Agent $agent The agent to filter by
+     *
+     * @throws NotFoundException if no related agent can be found
```

Comment:
I would not throw an exception in this case, but return an empty `Person` object instead.